Packs/AWS-SNS-Listener/Integrations/AWSSNSListener/AWSSNSListener.py

```diff
@@ -0,0 +1,268 @@
+from CommonServerPython import *  # noqa: F401
+from CommonServerUserPython import *
+from tempfile import NamedTemporaryFile
+from traceback import format_exc
+from collections import deque
+import uvicorn
+from secrets import compare_digest
+from fastapi import Depends, FastAPI, Request, Response, status
+from fastapi.security import HTTPBasic, HTTPBasicCredentials
+from fastapi.security.api_key import APIKeyHeader
+from fastapi.openapi.models import APIKey
+import base64
+from M2Crypto import X509
+
+
+PARAMS: dict = demisto.params()
+sample_events_to_store = deque(maxlen=20)  # type: ignore[var-annotated]
+
+app = FastAPI(docs_url=None, redoc_url=None, openapi_url=None)
+basic_auth = HTTPBasic(auto_error=False)
+token_auth = APIKeyHeader(auto_error=False, name='Authorization')
+
+
+PROXIES, USE_SSL = handle_proxy_for_long_running()
+
+
+class AWS_SNS_CLIENT(BaseClient):
+    def __init__(self, base_url):
+        is_proxy = False
```

Comment:
no need for is_proxy, you can either provide `bool(PROXIES)` to BaseClient init or use a walrus operator (i.e. is_proxy := bool(PROXIES)).